draft-ietf-quic-qpack.md

```diff
@@ -1199,6 +1199,12 @@ are registered in the "HTTP/3 Error Code" registry established in {{HTTP3}}.
 
 # Static Table
 
+This table was generated by analyzing actual internet traffic in 2018 and
+manually crafting the table for optimal compression of common headers.  Due to
+this methodology, some of the entries may be inconsistent.  Notably,
+"content-type: text/plain; charset=utf-8" appears twice with two different
+whitespace encodings (entries 52 and 54).
```

Comment:
```suggestion
this methodology, some of the entries may be inconsistent.
```

It's probably enough to observe that there are inconsistencies that derive from the source data.

Reply:
I feel like we've gotten a couple questions about this one in particular, but I can remove it.

Reply:
I would like to support @martinthomson because `text/plain; charsert=utf-8-` does not appear *twice*. The other is `text/html`.